src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/RequestSql/SqlRequestSettingsType.php

```diff
@@ -48,13 +49,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                 ],
                 'translation_domain' => false,
             ])
+            ->add('default_file_separator', TextType::class, [
+                'label' => $this->trans('Select your default file separator', 'Admin.Advparameters.Feature'),
+                'translation_domain' => false,
```

Comment:
```suggestion
                'translation_domain' => false,
                'constraints' => [
                    new NotBlank(),
                    new Length([
                        'max' => 1,
                        'maxMessage' => $this->trans(
                            'This field cannot be longer than %limit% characters.',
                            'Admin.Notifications.Error',
                            [
                                '%limit%' => 1,
                            ]
                        ),
                    ]),
                ],
```

Reply:
The controller do not validate the form, so the constraint won't be used.

Reply:
A select with `, ; | TAB` who use another separator 🤣

Reply:
No, we just need to validate the form in the controller, but it has been done weirdly and it makes it complicated.

Reply:
I'm going to answer in French and do a deepl translation

On peut toujours faire mieux mais quand ça fonctionne ça fonctionne si l'idée du select est validée, la QA pourrait inspecter le code remplacer la valeur du select et **encore** avoir une erreur. Je pense vraiment que toutes ces contraintes ne font pas avancer le projet.

Et d'un autre côté je trouve super que la QA teste tous les cas. Il faudrait un arbitrage sur ces PR bloqué pour des cas improbables et pourquoi pas les documenter dans avec un TODO beaucoup de personnes arrête après 1 2 review de la QA par manque de temps

--------------------------------------------------------------------------------------------------------------

You can always do better but when it works it works if the idea of the select is validated, the QA could inspect the code replace the value of the select and still have an error. I really think that all these constraints don't help the project.

On the other hand, I think it's great that the QA tests all the cases. There should be arbitration on these blocked PRs for unlikely cases and why not document them in a TODO? Many people stop after 1 2 reviews of the QA due to lack of time.

@kpodemski

Reply:
I would prefer having a select with limited number of options to make things easier

Reply:
In this case, I suggest this list : 
`,` `;` `\t` `|` ` `